src/steamship/app/invocable_response.py

```diff
@@ -121,7 +121,7 @@ def error(
         error: Optional[SteamshipError] = None,
         exception: Optional[Exception] = None,
         prefix: Optional[str] = None,
-    ) -> Response[T]:
```

Comment:
I personally like this naming a lot. It's a Response that Invocables will return.

Reply:
👍 to all the renames!